docs/OESDLBugBar.md

```diff
@@ -0,0 +1,86 @@
+# Open Enclave SDL Bug Bar
+
+The information listed in this “bug bar” is intended to help Open Enclave SDK developers to triage bugs and determine the bug severity in terms of security. 
+
+This bug bar and the ratings are derived based on [Microsoft SDL Bug bar](https://aka.ms/sdlbugbar) to apply for the enclaves, specifically to the enclaves created using Intel SGX technology. It is highly recommended to fix all known bugs with critical, important, or moderate severity prior to release. 
+
+The bug bar describes different severities for the bugs affecting enclaves. Enclaves are the privileged execution environment which is a security domain. There is a security boundary between enclave and rest of the system, this is a stronger security boundary that should be defended. It is important to understand the enclave security boundary, it should be apt to consider this as a confidentiality & integrity boundary. Anything that is inside enclave should never be accessible or modified from outside of the enclave. 
+
+Enclave security boundary is quite interesting, everything that is outside this boundary is adversarial which can fully control the existence of the enclave but can’t access or modify the data over inside the enclave crossing the enclave security boundary. Things that are outside the enclave security boundary are the current application, other enclaves, OS kernel, VMM/Hypervisor & SMM. Except the CPU everything is untrusted for enclaves. 
+
+
+## Enclave – Vulnerability Type Pivot 
```

Comment:
(same response as the previous) I thought about having this severity pivot instead vulnerability pivot. I feel it is easier to navigate by knowing the class of vulnerability and reaching to the severity than the other way, but it could be just me. If you insist I can change it to severity pivot or even have both.

Reply:
@CodeMonkeyLeet Based on your feedback I will change to severity pivot, do you suggest both severity pivot and vulnerability pivot or just severity pivot?